test/cql-pytest/test_tablets.py

```diff
@@ -52,3 +52,15 @@ def test_alter_cannot_change_vnodes_to_tablets(cql, skip_without_tablets):
     with new_test_keyspace(cql, ksdef) as keyspace:
         with pytest.raises(InvalidRequest):
             cql.execute(f"ALTER KEYSPACE {keyspace} WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 1}} AND tablets = {{'initial': 1}};")
+
+
+def test_alter_doesnt_enable_tablets(cql, skip_without_tablets):
+    ksdef = "WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};"
+    with new_test_keyspace(cql, ksdef) as keyspace:
+        cql.execute(f"ALTER KEYSPACE {keyspace} WITH replication = {{'class': 'NetworkTopologyStrategy'}};")
+
+        res = cql.execute(f"DESCRIBE KEYSPACE {keyspace}").one()
+        assert "NetworkTopologyStrategy" in res.create_statement
```

Comment:
After a recent patch, don't we have the "tablets" in the DESCRIBE KEYSPACE as well?
If we do, you can check that instead of the system_schema.scylla_keyspaces check below which I don't even know if we want to make official (did we document it anywhere?)

Reply:
>  After a recent patch, don't we have the "tablets" in the DESCRIBE KEYSPACE as well?

We do, but it's not yet merged :)

> If we do, you can check that instead of the system_schema.scylla_keyspaces check below which I don't even know if we want to make official (did we document it anywhere?)

It's documented in docs/cql/ddl.rst

Reply:
Ok then, then let's keep it like that for now.